app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -189,6 +196,9 @@ class FeedFragment : BaseListFragment<FeedState, Unit>() {
 
         empty_state_view?.let { animateView(it, false, 0) }
         animateView(error_panel, false, 0)
+        if (swipeRefreshLayout.isRefreshing){
+            swipeRefreshLayout.isRefreshing = false
+        }
```

Comment:
```suggestion
        swipeRefreshLayout.isRefreshing = false
```
If I'm not mistaken, there's no difference.

Reply:
But there is a difference of checking if currently there is a refreshing action and set that action false otherwise dont. I thought this is a healthier way of doing like when you dismiss a progressDialog.

Reply:
In your original code, `isRefreshing` will be set to `false` if it's `true`, otherwise it stays the same. This is the same as setting it to `false` regardless of its value.

Reply:
I will try the change your recommended, if it doesn't change any behavior I will just commit it.